relocation.bs

```diff
@@ -261,7 +261,8 @@ We believe a language solution is best suited here:
 
 This paper suggests to introduce a new keyword, named `reloc`. `reloc` acts as a
 unary operator that can be applied to named, local, complete objects (in other
-words: local function variables).
+words: function-local non-static variables and, with some restrictions,
+parameters and structured bindings).
```

Comment:
Not so sure about the wording here. Technically, structured bindings cannot be relocated. The structured binding declaration gets promoted to a "structured relocation" declaration, where each identifier is a complete object on its own and is no longer a structured binding.